pyadjoint/tape.py

```diff
@@ -101,15 +125,7 @@ def __exit__(self, *args):
                     var.create_block_variable()
 
 
-def no_annotations(function):
-    """Decorator to turn off annotation for the decorated function."""
-
-    @wraps(function)
-    def wrapper(*args, **kwargs):
-        with stop_annotating():
-            return function(*args, **kwargs)
-
-    return wrapper
+no_annotations = stop_annotating()
```

Comment:
```suggestion
no_annotations = stop_annotating()
"""Decorator to turn off annotation for the decorated function."""
```

Reply:
I did not know you could do that, that's very useful.